src/AppInstallerCLICore/Commands/UninstallCommand.cpp

```diff
@@ -10,6 +10,7 @@
 
 using AppInstaller::CLI::Execution::Args;
 using AppInstaller::CLI::Workflow::ExecutionStage;
+using AppInstaller::CLI::Workflow::SearchPurpose;
```

Comment:
nit:
using AppInstaller::CLI::Workflow::ExecutionStage;
using AppInstaller::CLI::Workflow::SearchPurpose;

could just be 

using AppInstaller::CLI::Workflow;

Reply:
Will change; I was a bit confused as to which was proper since I saw differing patterns in different places